src/main/groovy/edu/oregonstate/mist/cardsapi/resources/CardsResource.groovy

```diff
@@ -168,6 +168,21 @@ class CardsResource extends Resource {
                     "must match pattern: " + validPattern).build()
         }
 
+        // Validate energy
+        if(energyMin.present && energyMax.present &&
+                !(energyMin.get() >= 0 && energyMin.get() <= 999 &&
+                        energyMax.get() >= 0 && energyMax.get() <= 999)) {
```

Comment:
The indentation here looks a bit odd and confusing. Could you align these conditions? For example: https://stackoverflow.com/questions/19825746/whats-the-proper-way-to-format-if-statements-in-java-with-multiline-ands-or